package.json

```diff
@@ -2,7 +2,7 @@
   "name": "atom-icons",
   "displayName": "Atom Icons",
   "description": "Atom file and folder icons for VS Code",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
No need to increment the version in this PR, I'll do it when I make the release for this!

Reply:
It's fixed.